src/odin/notifications/mod.rs

```diff
@@ -51,6 +52,39 @@ fn is_webhook_enabled() -> bool {
   false
 }
 
+fn is_webhook_include_public_ip() -> bool {
+  if fetch_var(WEBHOOK_INCLUDE_PUBLIC_IP, "0")
+    .trim_start_matches('"')
+    .trim_end_matches('"')
+    .eq("1")
+  {
+    debug!("Webhook Include Public IP found!");
+    return true;
+  }
+  false
+}
+
+#[tokio::main]
+async fn get_public_ip() -> String {
+  if let Some(ip) = public_ip::addr().await {
+    return ip.to_string();
+  }
+  "Couldn't determine Public IP address.".to_string()
+}
+
+pub fn get_notification_server_name() -> String {
+  if is_webhook_include_public_ip() {
+    return [
+      get_server_name(),
+      " (".to_string(),
+      get_public_ip(),
+      ")".to_string(),
+    ]
+    .join("");
+  }
+  get_server_name()
+}
+
```

Comment:
Lets try and rip tokio out of there please, thats a large lib to include just for one http request. Use the already included lib of reqwest via something like this:

```rust

let body = reqwest::blocking::get("https://api.ipify.org?format=json").unwrap()
    .text().unwrap();
```